core/ledger/kvledger/tests/env.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/hyperledger/fabric/msp"
 	"github.com/hyperledger/fabric/msp/mgmt"
 	"github.com/hyperledger/fabric/protoutil"
+	"github.com/spf13/viper"
```

Comment:
see my other comment, but do we really need to use viper?   I actually worked pretty hard to remove viper last year

Reply:
sure we can remove it from this file, but anything that uses msp/mgmt will still use viper in its dependencies (`LoadLocalMSP`, etc)

Reply:
agreed .... someday we will get to that as well.  thx